fern/docs/pages/installation/installation.mdx

```diff
@@ -61,46 +80,49 @@ You need to choose one option per category (LLM, Embeddings, Vector Stores, UI).
 
 #### LLM
 
-| **Option**   | **Description**                                                        | **Extra**           |
-|--------------|------------------------------------------------------------------------|---------------------|
-| **ollama**   | Adds support for Ollama LLM, requires Ollama running locally           | llms-ollama         |
-| llama-cpp    | Adds support for local LLM using LlamaCPP                              | llms-llama-cpp      |
-| sagemaker    | Adds support for Amazon Sagemaker LLM, requires Sagemaker endpoints    | llms-sagemaker      |
-| openai       | Adds support for OpenAI LLM, requires OpenAI API key                   | llms-openai         |
-| openailike   | Adds support for 3rd party LLM providers compatible with OpenAI's API  | llms-openai-like    |
-| azopenai     | Adds support for Azure OpenAI LLM, requires Azure endpoints            | llms-azopenai       |
-| gemini       | Adds support for Gemini LLM, requires Gemini API key                   | llms-gemini         |
+| **Option** | **Description**                                                       | **Extra**        |
+| ---------- | --------------------------------------------------------------------- | ---------------- |
+| **ollama** | Adds support for Ollama LLM, requires Ollama running locally          | llms-ollama      |
+| llama-cpp  | Adds support for local LLM using LlamaCPP                             | llms-llama-cpp   |
+| sagemaker  | Adds support for Amazon Sagemaker LLM, requires Sagemaker endpoints   | llms-sagemaker   |
+| openai     | Adds support for OpenAI LLM, requires OpenAI API key                  | llms-openai      |
+| openailike | Adds support for 3rd party LLM providers compatible with OpenAI's API | llms-openai-like |
+| azopenai   | Adds support for Azure OpenAI LLM, requires Azure endpoints           | llms-azopenai    |
+| gemini     | Adds support for Gemini LLM, requires Gemini API key                  | llms-gemini      |
 
```

Comment:
Why didn't you add fireworks here?

Reply:
i have added fireworks installation steps near open-ai